src/flownet/config_parser/_config_parser.py

```diff
@@ -437,6 +437,11 @@ def _to_abs_path(path: Optional[str]) -> str:
                         MK.Description: "Number of points along a tube to check whether they are in"
                         "non reservoir for removal purposes.",
                     },
+                    "min_permeability": {
+                        MK.Type: types.Number,
+                        MK.AllowNone: True,
+                        MK.Description: "Minimum allowed permeability in mD before a tube is removed.",
+                    },
```

Comment:
Suggestion: `"(...) before a tube is removed (i.e., its cells are made inactive)."`

Plus, would we like to have this to be a different value across e.g. possible different layer zones?

Reply:
> Suggestion: `"(...) before a tube is removed (i.e., its cells are made inactive)."`

Done ✔️ 

> Plus, would we like to have this to be a different value across e.g. possible different layer zones?

Maybe, but I think we should first test whether this has a positive impact before making things more complex.